lib/omnibus/software.rb

```diff
@@ -571,11 +571,17 @@ def with_standard_compiler_flags(env = {}, opts = {})
           freebsd_flags
         when "windows"
           arch_flag = windows_arch_i386? ? "-m32" : "-m64"
-          opt_flag = windows_arch_i386? ? "-march=i686" : "-march=x86-64"
+          #opt_flag = windows_arch_i386? ? "-march=i686" : "-march=x86-64"
```

Comment:
It was overriding openssl's explicit -march i486 flag.  I'll add a comment to that effect too.

@jaym I was leaving the code in for context as to what cannot be done without investigation.  It's easy for someone to accidentally enable a flag without checking history to see if that flag was disabled.